doajtest/unit/resources/crossref_article_uploads.xml

```diff
@@ -0,0 +1,1414 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<doi_batch version="4.4.2" xmlns="http://www.crossref.org/schema/4.4.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.crossref.org/schema/4.4.2" xmlns:jats="http://www.ncbi.nlm.nih.gov/JATS1">
+    <head>
+        <doi_batch_id>1dbb27d1030c6c9d9d-7ff0</doi_batch_id>
+        <timestamp>200504260247</timestamp>
+        <depositor>
+            <depositor_name>your name</depositor_name>
+            <email_address>your@email.com</email_address>
+        </depositor>
+        <registrant>WEB-FORM</registrant>
+    </head>
+    <body>
```

Comment:
One thing we should do is check exactly what format the XML uploads are going to be.  Is this document taken from the Crossref XML API?  It's possible that users will unwrap the feed XML (maybe up to and including this `body` tag) before they give us the data.  It would be good if we could handle both situations (which your code might do, I haven't worked that out yet ;) )

Reply:
>  It's possible that users will unwrap the feed XML (maybe up to and including this body tag) before they give us the data.
I don't think I understand. Could you explain please? XML File Upload Form requires information which schema the file should be validated against. Crossref schema requires <doi_batch> tag as a file root.

Reply:
ah perfect, in which case that's all good.   I wasn't sure if the `head` and `body` bits were related to the API format.